components/BlogPost/index.tsx

```diff
@@ -0,0 +1,33 @@
+import MarkdownRenderer from '../MarkdownRenderer';
+
+import styles from './style.module.css';
+
+
+import { CalendarOutlined } from  '@ant-design/icons';
+
+const BlogPost = (props: { data: { title: string; authorImage: string; author: string; date: string }; content: string; }) => {
+    return (
+        <div className="container mx-auto">      
+            <div className={styles.blogContainer}>
+                <div className={styles.header}>
+                    <h1 className={styles.title}>{props.data.title}</h1>
```

Comment:
```suggestion
                    <h1 className={styles.title}>{title}</h1>
```

This is an example to all others.